pkg/operator/management/management_state.go

```diff
@@ -67,3 +70,34 @@ func IsOperatorManaged(state v1.ManagementState) bool {
 	}
 	return true
 }
+
+// SyncAction tells caller of GetSyncAction what to do on operator Sync() based on the management state
+type SyncAction string
+
+const (
+	// SyncActionManage means the operator should manage its operands (create them if missing, update when needed).
+	SyncActionManage = "Manage"
+	// SyncActionDelete means the operator should delete its operands.
+	SyncActionDelete = "Delete"
```

Comment:
nit: prefer names to match the values set in the API.

Reply:
Done